### ISSUE TITLE
Get notification of any new comment on any post

### ISSUE THREAD

**Issue by justinkelly** [eduardoboucas/staticman#122]
Hi Staticman,

I've setup staticman with jekyll and mailgun and its working great !!! - thanks!!!!

1 thing I haven't figured out yet is how (as blog admin) to get notifications of all new comments on any post
- without having to manually create a test comment and subscribe on each blog post

Is this possible?

Cheers

Justin

**Comment by binarymist** [comment]
Yes, you should be sent an email from github for each pull request as they come in, just make sure you have pull requests setup.

**Comment by justinkelly** [comment]
Ok thanks - i now get the emails from github (ive set moderation to false)

just is a bit of a pain as the github emails obviously do not have the url of the post the comment is for

where as the reply notifications from staticman do have the link

the link makes it easier to see the full conversation etc..

**Comment by binarymist** [comment]
`moderation` needs to be set to true: https://github.com/binarymist/BinaryMistBlog/blob/666bd4f893f3fb5b90535ab985a5346882705c4b/staticman.yml#L47

The github email has a link to the Pull Request which has the name of the post, the github email also has the Commit Summary which has the name of the post.

The reply notifications have a link, but you have to click it to find out the name of the post, so this takes one click longer.

I don't think this is an issue is it? Notifications do come through, in fact I get several of them, a bit I think: #182 but am waiting for some guidance from @eduardoboucas?
